### PR TITLE
update style to more closely follow java standards

### PR DESCRIPTION
**Pull request by noonhub**

### CODE REVIEW

File: src/main/java/com/asana/examples/ExampleCreateProjectAndStreamEvents.java [hunk 1]
```diff
@@ -8,16 +8,13 @@
 import java.util.List;
 
 /**
- *
  * API Key Instructions:
- *
+ * <p/>
```
Comment (epelz) [inline src/main/java/com/asana/examples/ExampleCreateProjectAndStreamEvents.java]: I think this snuck in?

File: src/main/java/com/asana/examples/ExampleCreateProjectAndStreamEvents.java [hunk 2]
```diff
@@ -27,7 +24,7 @@ public static void main( String[] args) throws Exception
 
         // find your "Personal Projects" project
         Workspace personalProjects = null;
-        for (Workspace workspace: client.workspaces.findAll()) {
+        for (Workspace workspace : client.workspaces.findAll()) {
```
Comment (epelz) [inline src/main/java/com/asana/examples/ExampleCreateProjectAndStreamEvents.java]: Is this style common in Java?

File: src/main/java/com/asana/examples/ExampleCreateTaskAndUpload.java [hunk 3]
```diff
@@ -12,16 +12,13 @@
 import java.util.List;
 
 /**
- *
  * API Key Instructions:
- *
+ * <p/>
```
Comment (epelz) [inline src/main/java/com/asana/examples/ExampleCreateTaskAndUpload.java]: this too?

File: src/main/java/com/asana/examples/ExampleOAuth.java [hunk 4]
```diff
@@ -10,18 +10,14 @@
 import java.net.URI;
 
 /**
- *
  * OAuth Instructions:
- *
+ * <p/>
```
Comment (epelz) [inline src/main/java/com/asana/examples/ExampleOAuth.java]: This too?

File: src/main/java/com/asana/examples/ExamplePersonalAccessToken.java [hunk 5]
```diff
@@ -4,16 +4,13 @@
 import com.asana.models.User;
 
 /**
- *
  * API Key Instructions:
- *
+ * <p/>
```
Comment (epelz) [inline src/main/java/com/asana/examples/ExamplePersonalAccessToken.java]: This too?

File: src/main/java/com/asana/examples/ExampleServer.java [hunk 6]
```diff
@@ -12,22 +12,18 @@
 import java.util.UUID;
 
 /**
- *
  * OAuth Instructions:
- *
+ * <p/>
```
Comment (epelz) [inline src/main/java/com/asana/examples/ExampleServer.java]: This too?

**Comment by epelz** [comment]
Looks good, other than a few small style nits. Want to fix those before I merge?

File: src/main/java/com/asana/examples/ExampleCreateProjectAndStreamEvents.java (hunk 2 above)
Comment (noonhub) [inline src/main/java/com/asana/examples/ExampleCreateProjectAndStreamEvents.java]: Yes, this is proper style.

File: src/main/java/com/asana/examples/ExampleCreateProjectAndStreamEvents.java (hunk 1 above)
Comment (noonhub) [inline src/main/java/com/asana/examples/ExampleCreateProjectAndStreamEvents.java]: This is debatable but technically correct. Down with taking it out. http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#format

**Comment by noonhub** [comment]
I think this is ready to merge.

**Pull request merged**